src/screens/ConnectSignUpScreen.js

```diff
@@ -75,6 +75,8 @@ export default class ConnectSignUp extends React.Component {
       militaryBranch: t.enums(this.state.militaryBranches),
       unit: t.String,
       notes: t.String,
+      phoneNumber: t.String,
```

Comment:
you might want to look into a different input format for this field. We use a library called `tcomb-form-native`, which may have some support for phone number inputs: 

https://github.com/gcanti/tcomb-form-native